src/peer-connection-wrapper.cpp

```diff
@@ -520,7 +520,7 @@ Napi::Value PeerConnectionWrapper::createDataChannel(const Napi::CallbackInfo &i
                     init.reliability.rexmit = reliability.Get("rexmit").As<Napi::Number>();
```

Comment:
This could also be fixed in the same way for the sake of consistency.
```suggestion
                    init.reliability.rexmit = reliability.Get("rexmit").As<Napi::Number>().Uint32Value();
```